java-spiffe-core/grpc-netty-macos-aarch64/build.gradle

```diff
@@ -0,0 +1,12 @@
+description = "Java SPIFFE Library GRPC-Netty MacOS module"
+
+dependencies {
+    implementation group: 'io.grpc', name: 'grpc-netty', version: "${grpcVersion}"
+
+    // version must match the one in grpc-netty
+    implementation group: 'io.netty', name: 'netty-transport-native-kqueue', version: '4.1.79.Final', classifier: 'osx-aarch_64'
```

Comment:
This needs to match the transient `io.netty` dependencies. Maybe consider using the netty-bom instead to pin all versions? Sadly the transient version is also not up-to-date.

Reply:
Can we extract the Netty version to a variable in `java-spiffe-core/build.gradle` so that we can control the version in one place for both `java-spiffe-core/grpc-netty-macos` and `java-spiffe-core/grpc-netty-macos-aarch64`?

You can find some examples of how to do this in the root `build.gradle`, e.g. `grpcVersion`.